include/mbedtls/rsa.h

```diff
@@ -238,15 +238,22 @@ int mbedtls_rsa_import_raw( mbedtls_rsa_context *ctx,
  *                     - Derive P, Q from N, D, E
  *                     - Derive N, D from P, Q, E.
  *
- *                 - Alternative implementations need not support these
- *                   and may return \c MBEDTLS_ERR_RSA_BAD_INPUT_DATA instead.
+ *                 - Alternative implementations need not support all of the
+ *                   above private key derivations; further, they might put
+ *                   constraints on acceptable sizes and values of the imported
+ *                   parameters. In case a failure originating from these
+ *                   implementation-specific restrictions, the error code
+ *                   \c MBEDTLS_ERR_RSA_UNSUPPORTED_OPERATION should be
```

Comment:
There's a clear subject here, so please avoid the passive voice: “In case a failure originating from these implementation-specific restrictions, alternatives implementations should return the error code MBEDTLS_ERR_RSA_UNSUPPORTED_OPERATION”.

(I'd even say “shall”, not “should”, since we're mandating it and not just recommending. But that's a departure from our usual style, so I guess it should stay “should”.)